x/stake/keeper/validator.go

```diff
@@ -425,7 +425,8 @@ func (k Keeper) UpdateBondedValidators(
 				}
 				newValidatorBonded = true
 			}
-
+		} else {
+			// TODO: document why we must break here.
```

Comment:
Why must we break here? Was this an accidental merge change?

Reply:
Ah, because we changed the logic to break as soon as we hit a revoked validator.